### DISCUSSION TITLE
Difficulty Capturing Screenshots and Downloading Content from nooriacademy.com

### DISCUSSION THREAD

**Discussion by TechyOmi** [ReVanced/revanced-discussions#1094] [Question] [answered]
I am having **trouble taking screenshots and downloading PDFs and videos** from the website  [](https://web.nooriacademy.com/). While I am able to read the PDFs on the site, I cannot capture screenshots or download any of the content.

Could you please provide any advice or recommendations on how I can overcome these limitations and access the downloadable content on this site? I'm open to using any tools, plugins, or methods that would grant me the ability to screenshot and download the materials I need from Nooriacademy.com.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
ReVanced cannot help or do anything with viewing websites.
